apps/api-documenter/src/nodes/DocHeading.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import {
+  IDocNodeParameters,
+  DocNode
+} from '@microsoft/tsdoc';
+import { CustomDocNodeKind } from './CustomDocNodeKind';
+
+/**
+ * Constructor parameters for {@link DocHeading}.
+ */
+export interface IDocHeadingParameters extends IDocNodeParameters {
+  title: string;
+  level?: number;
+}
+
+/**
+ * Represents table, similar to an HTML `<h1>` or `<h2>` element.
+ */
+export class DocHeading extends DocNode {
+  public readonly title: string;
+  public readonly level: number;
+
+  /**
+   * Don't call this directly.  Instead use {@link TSDocParser}
+   * @internal
+   */
+  public constructor(parameters: IDocHeadingParameters) {
+    super(parameters);
+    this.title = parameters.title;
+    this.level = parameters.level !== undefined ? parameters.level : 1;
+
+    if (this.level < 1 || this.level > 5) {
```

Comment:
>this.level > 5 [](start = 26, length = 14)

Is there a reason the upper-bound is 5 not 6? I thought h1...h6 were valid HTML.

[https://www.w3.org/MarkUp/html3/headings.html](https://www.w3.org/MarkUp/html3/headings.html)

Reply:
Personal preference I guess?  5 nested headers already seems like a sure ticket to graphic design "time out".

---
In reply to: [236555749](https://github.com/Microsoft/web-build-tools/pull/952#discussion_r236555749) [](ancestors = 236555749)